modules/tinymce/src/core/main/ts/api/EditorOptions.ts

```diff
@@ -52,9 +52,37 @@ export interface OptionSpec<T, U> extends BaseOptionSpec {
 }
 
 /**
- * TinyMCE Editor Options API.
+ * TinyMCE Editor Options API. The options API provides the ability to register, lookup and set editor options.
+ *
+ * @summary All options need to be registered before they can be used in the editor. This is done by using the `register()` API which requires a name
+ * and an option specification. The specification should contain a `processor` and an optional `default` value. The processor is used to parse
+ * and validate the option value either passed in the initial configuration or via the `set()` API.
+ * <br><br>
```

Comment:
Why do we need BRs here wouldn't newlines be converted by moxiedoc to newlines in asciidoc? Should the summary be rendered as pre elements instead not sure where the issue lies on the site templates, asciidoc or moxiedoc?

Reply:
No, because it works the same way that the old Markdown (tinymcenext) version did it removes all newlines similar to how it's rendered in HTML. I'd like to change that to be honest, but that's going to require changes to be made to 5.x as well.